lib/l4p/udp_stream.c

```diff
@@ -276,6 +276,74 @@ tle_udp_stream_open(struct tle_ctx *ctx,
 	return &s->s;
 }
 
+	struct tle_stream *
```

Comment:
The breaking of this definition seems wonky...

Reply:
Their style...not mine.

Reply:
the leading space will be removed